src/components/atoms/TokenDetail/tokenDetail.atom.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { ITokenDetail } from "./tokenDetail.atom.types";
+import styles from "./tokenDetail.atom.module.scss";
+import Typography from "../Typography/typography.atom";
+import {EFontWeight} from "../../../shared/types/styles.types";
+import Link from "../Link/link.atom";
+
+const TokenDetail: React.FC<ITokenDetail> = (props) => {
+  const {assetType, logo: Logo, name, purchaseLink} = props;
+  
+  return (
+    <div className={styles.tokenDetail}>
+      <div>
+        <Logo className={styles.logo} /> 
+      </div>
+      <div className={styles.tokenLabel}>
+        <Typography
+          fontSize={16}
+          fontWeight={EFontWeight.BOLD}
+        >
+          {name}
+        </Typography>
+        {purchaseLink && <Link className={styles.purchaseLink} link={purchaseLink}>{assetType}</Link>}
```

Comment:
We should stop click propagation when click on the link so that the dropdown doesn't open. We can do this inside the Link component with onClick listener maybe?